specification/v1.1.0-rc/OSDM-online-api-v1.1.0-rc.yml

```diff
@@ -3095,6 +3118,21 @@ components:
           type: string
         carrierConstraintText:
           type: string
+        combinationTags:
```

Comment:
at offer level,  I called this tripTag, since the returnTag is a kind of combinationtag as well, IMO. But I am fine with the combinationTag naming. However I wonder: the tag on product level is IMO more as an infomation, and the one(s) on offer level those to be actually used when putting an offer together for the the upper level. What do you think ?